src/App.css

```diff
@@ -1,38 +1,26 @@
 .App {
```

Comment:
put this one to lower case, use kebab-case for naming of css classes, like you did in the rest of the file.